cloud/pkg/cloudhub/channelq/channelq.go

```diff
@@ -1,13 +1,14 @@
 package channelq
 
 import (
+	"context"
```

Comment:
Instead of importing context besides beehivecontext, I'd suggest include golang context functionalities into beehivecontext. To keep channelq only rely on beehive.